tests/sparseml/export/transformers/test_generation_export.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shutil
+import unittest
+from pathlib import Path
+
+import pytest
+
+from huggingface_hub import snapshot_download
+from parameterized import parameterized_class
+from sparseml import export
+from tests.testing_utils import parse_params
+
+
+CONFIGS_DIRECTORY = "tests/sparseml/export/transformers/generation_configs"
```

Comment:
any reason we wouldn't want to use relative path here? hardcoding this would require us to always run from `sparseml/` not that this is not always the case though...

Reply:
yah I kept the path from `sparseml` as that is the directory uses to run all gha tests